ReactSkia/RNInstance.cpp

```diff
@@ -158,6 +166,7 @@ xplat::module::CxxModule* RNInstance::moduleForName(std::string moduleName) {
 
 void RNInstance::InitializeJSCore() {
   instance_ = std::make_shared<Instance>();
+  componentViewRegistry_ = std::make_unique<ComponentViewRegistry>();
```

Comment:
May be we have to register components before JSCore initialisation

Reply:
addressed